File: 이상한_문자_만들기.cpp
#include <iostream>
using namespace std;

string solution(string s) {
	int index = 0;
	for (int i = 0; i < s.length(); i++) {
		if (s[i] == ' ') index = 0;
		else {
			if (index % 2) s[i] = tolower(s[i]);
			else s[i] = toupper(s[i]);
			index++;
		}
	}

	return s;
}

File: 이상한_문자_만들기.py
def solution(s):
    result = ""
    index = 0
    for i in range(len(s)):
        if s[i] == ' ':
            index = 0
            result += ' '
        else:
            if index % 2:
                result += s[i].lower()
            else:
                result += s[i].upper()
            index += 1
            
    return result
